addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/bbqtemperature.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="bbqtemperature">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+			<bridge-type-ref id="tcpbridge" />
+			<bridge-type-ref id="RFXtrx433" />
+			<bridge-type-ref id="RFXrec433" />
+		</supported-bridge-type-refs>
+
+		<label>RFXCOM BBQ Temperature Sensor</label>
+		<description>A BBQ Temperature device.</description>
+
+		<channels>
+			<channel id="foodTemperature" typeId="temperature" />
+			<channel id="bbqTemperature" typeId="temperature" />
```

Comment:
@kaikreuzer everything is fine now, the only possible issue is about reusing of channels, but that could of course be separated. Can you check that?